openbas-front/src/admin/components/atomic_testings/atomic_testing/AtomicTestingHeader.tsx

```diff
@@ -65,8 +72,10 @@ const AtomicTestingHeader = () => {
         <Typography variant="h1" gutterBottom classes={{ root: classes.title }}>
           {atomic.atomic_title}
         </Typography>
+        {atomicDetail?.status_label
+          && <StatusChip status={atomicDetail?.status_label} />
```

Comment:
Like discussed pass status label in the main dto or not use a atomicdetail dto at all